packages/turf-line-to-polygon/index.ts

```diff
@@ -29,16 +31,23 @@ function lineToPolygon<G extends LineString|MultiLineString>(
         properties?: Properties,
         autoComplete?: boolean,
         orderCoords?: boolean,
+        mutate?: false,
```

Comment:
Shouldn't this have a type of `boolean`? Or you could get rid of line 46 and do `mutate = false`.

Reply:
Yes it should of course be boolean, it should have been fixed now. For consistency with the rest of the modules code base I think line 46 should stay, even though I do find `mutate = false` to be more readable.